project-base/src/Shopsys/ShopBundle/Model/Order/Item/OrderItem.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Model\Order\Item;
+
+use Doctrine\ORM\Mapping as ORM;
+use Shopsys\FrameworkBundle\Model\Order\Item\OrderItem as BaseOrderItem;
+use Shopsys\FrameworkBundle\Model\Order\Order as BaseOrder;
+use Shopsys\FrameworkBundle\Model\Pricing\Price;
+
+/**
+ * @ORM\Table(name="order_items")
+ * @ORM\Entity
+ */
+class OrderItem extends BaseOrderItem
+{
+    /**
+     * @param \Shopsys\ShopBundle\Model\Order\Order $order
+     * @param string $name
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Price $price
+     * @param string $vatPercent
+     * @param int $quantity
+     * @param string $type
+     * @param null|string $unitName
+     * @param null|string $catnum
+     */
+    public function __construct(
```

Comment:
This is not necessary here

Reply:
Like a whole constructor?
True, but we have constructors like this in every other extended entity.

Or?

Reply:
Oh, it is true, it is ok then